src/T2DMap.h

```diff
@@ -222,6 +220,9 @@ public slots:
     // room listing/selection widget, and by what,
     // as we now show room names (if present) as well.
     bool mIsSelectionUsingNames;
+    // The initialisation cannot be completed until both Host and TMap classes
+    // are setup - this gets set once init() has been done:
+    bool mIsInitialised;
```

Comment:
I know you like your `is` prefix, but isn't `if initialised then` a better read than `if is initialised then`?

Reply:
... that is slightly out of context isn't it, 😈  - `if (isInitialised) {` is no more or less hard to read and in other places it reminds (me at least) that it is a C++ `bool` and not a [C `int`](https://stackoverflow.com/q/1608318/4805858){are my roots showing 😟?}